nomad/structs/structs.go

```diff
@@ -8394,6 +8394,9 @@ type Evaluation struct {
 	// Raft Indexes
 	CreateIndex uint64
 	ModifyIndex uint64
+
+	CreateTime int64
+	ModifyTime int64
```

Comment:
You'll need to modify the methods here - https://github.com/hashicorp/nomad/blob/675f750038d2ddee003bc6a2efdef2c50b5f4907/nomad/structs/structs.go#L8522 that create different types of evals, to also include a create and modify timestamp.

Reply:
:+1: